site/app/views/submission/HomeworkView.php

```diff
@@ -444,23 +496,30 @@ function makeSubmission(user_id, highest_version, is_pdf, path, count) {
                     foreach ($files as $filename => $details) {
                         $clean_timestamp = str_replace("_", " ", $timestamp);
                         $path = $details["path"];
-                        if (strpos($filename, 'cover') == false) {
-                            // add each file that is not a cover to count_array 
-                            // each entry is in format timestamp/filename
-                            $count_array[$count] = $timestamp."/".$filename;
+                        if (strpos($filename, "cover") == false) {
```

Comment:
use `=== false` here as strpos can return `false` (string not found) or `0` (string at beginning) which are both `== false`.